src/labels/CustomLabels.labels

```diff
@@ -1485,6 +1485,38 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <protected>true</protected>
         <shortDescription>Processing</shortDescription>
         <value>Processing</value>
+    </labels>
+    <labels>
+        <fullName>RD2_ActiveSchedulesCurrentSchedule</fullName>
+        <categories>Recurring-Donations, Component</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Details of the schedule which is currently in effect</shortDescription>
+        <value>Current Schedule</value>
```

Comment:
@sam-knox , can you please review new labels?

Reply:
@sam-knox @RadSFDeveloper It might be good to hold review of new labels until we are officially ready for review.  There may still be changes.

Reply:
@npsp-reedestockton - Is this ready for review or should I wait?